### PR TITLE
LPS-144859 Update tests for when an objectField is updated

### PR DESCRIPTION
**Pull request by mateussgn**
Related issue: https://issues.liferay.com/browse/LPS-144859

### CODE REVIEW

**Comment by mateussgn** [comment]
Blocker Fix sent to bChan here: https://github.com/brianchandotcom/liferay-portal/pull/111808
I've sent it here to let CI run.

**Comment by mateussgn** [comment]
ci:test:relevant

**Comment by mateussgn** [comment]
ci:stop

**Comment by mateussgn** [comment]
ci:test:sf

**Comment by mateussgn** [comment]
ci:test:relevant

File: modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectFieldLocalServiceTest.java [hunk 1]
```diff
@@ -471,6 +471,7 @@ public void testUpdateCustomObjectField() throws Exception {
 
 		Assert.assertFalse(objectField.isIndexed());
 		Assert.assertTrue(objectField.isIndexedAsKeyword());
+		Assert.assertEquals("able_", objectField.getDBColumnName());
```
Comment (gabrielwas) [inline modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectFieldLocalServiceTest.java:474]: @mateussgn can you sort this assert and the similar ones? Thx

**Comment by mateussgn** [comment]
ci:forward

**Pull request closed without merging**